python-threatexchange/threatexchange/signal_type/pdq_index.py

```diff
@@ -11,21 +11,33 @@
 import pickle
 
 from threatexchange.signal_type.index import SignalTypeIndex, IndexMatch, T as IndexT
-from threatexchange.hashing.pdq_faiss_matcher import PDQMultiHashIndex
+from threatexchange.hashing.pdq_faiss_matcher import (
+    PDQMultiHashIndex,
+    PDQFlatHashIndex,
+    PDQHashIndex,
+)
 
 
 class PDQIndex(SignalTypeIndex):
     """
-    Wrapper around the pdq faiss index lib
+    Wrapper around the pdq faiss index lib using PDQMultiHashIndex
     """
 
     PDQ_CONFIDENT_MATCH_THRESHOLD = 31
     T = IndexT
 
+    @classmethod
+    def get_match_threshold(cls):
+        return cls.PDQ_CONFIDENT_MATCH_THRESHOLD
```

Comment:
(nit): Unless that value is reused somewhere else, feel free to inline and prevent a class constant from being created. (is there another one inside SignalType as well?)